src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -76,10 +89,16 @@ export default function DownloadListTable() {
       return 'Only Ineligible'
     }
   }
-  const removeItem = async (item: DownloadListItem) => {
+  const removeItem = async (item: DownloadListItem, fileName: string) => {
     try {
       await SynapseClient.removeItemFromDownloadListV2(item, accessToken)
-      refetch()
+      displayToast(
```

Comment:
Display the toast whenever an item is removed from the download list (on direct download or if an item was explicitly removed).